### PR TITLE
Fix style bindings mixin not observing changes after init

### PR DESCRIPTION
**Pull request by Dylnuge**
@azirbel @Addepar/uie 

If when extending the ember-charts library a user attempts to add or remove
style bindings, these changes will now be reflected.

### CODE REVIEW

**Comment by Dylnuge** [comment]
Need to update this in https://github.com/Addepar/ember-charts/blob/master/dependencies/ember-addepar-mixins/style_bindings.js for the same reasons

File: dependencies/ember-addepar-mixins/style_bindings.js [hunk 1]
```diff
@@ -15,25 +15,22 @@ Ember.AddeparMixins.StyleBindingsMixin = Ember.Mixin.create({
...
     styleBindings.forEach(function(binding) {
-      var property, style, tmp;
-      tmp = binding.split(':');
-      property = tmp[0];
-      style = tmp[1];
-      lookup[style || property] = property;
-    });
-    styles = Ember.keys(lookup);
-    properties = styles.map(function(style) {
-      return lookup[style];
```
Comment (Dylnuge) [inline dependencies/ember-addepar-mixins/style_bindings.js]: Rebase issue here reverted formatting to the old style. I'll fix this.

**Comment by azirbel** [comment]
bonus points if you make an `ember-addepar-mixins` repo that everyone can pull in as a dependency

**Comment by azirbel** [comment]
that's been on the wishlist for a while

File: dependencies/ember-addepar-mixins/style_bindings.js [hunk 2]
```diff
@@ -15,16 +15,17 @@ Ember.AddeparMixins.StyleBindingsMixin = Ember.Mixin.create({
...
       _this = this;
-    styleBindings = this.styleBindings;
+    styleBindings = this.get('styleBindings');
     if (!styleBindings) {
       return;
     }
     lookup = {};
     styleBindings.forEach(function(binding) {
-      var property, style, tmp;
+      var property, style, _ref;
```
Comment (azirbel) [inline dependencies/ember-addepar-mixins/style_bindings.js]: this doesn't look right

Comment (Dylnuge) [inline dependencies/ember-addepar-mixins/style_bindings.js]: You're fast! I just fixed it ;)

**Comment by azirbel** [comment]
LGTM, awesome :+1:

**Comment by Dylnuge** [comment]
Bonus points sound like something worth doing, I'll go ahead :smile:

**Comment by Dylnuge** [comment]
I'm going to merge this for now, but the `ember-addepar-mixins` repo is in the works :smile:

**Pull request merged**